src/main/java/org/mapfish/print/map/readers/MapReader.java

```diff
@@ -28,9 +28,11 @@
  */
 public abstract class MapReader {
     protected final float opacity;
+    protected String pdfLayerName;
 
     public MapReader(PJsonObject params) {
         opacity = params.optFloat("opacity", 1.0F);
+        pdfLayerName = params.optString("pdfLayerName","");
```

Comment:
So I am looking specifically to document `pdfLayerName` here https://github.com/mapfish/mapfish-print-v2/blob/main/docs/configuration.rst in this file as part of the PR?

Even if this page is incomplete presently, by documenting new stuff we help prevent having the problem be worse.